src/components/DisplayToy.js

```diff
@@ -13,17 +13,15 @@ const DisplayToy = (props) => {
 
   const history = useHistory();
 
-  const handleDelete = (toyId) => {
-    dispatch(deleteToy(toyId, history));
-  };
-
   return (
     <Container>
       <div>
         <h1 className="header text-center">{toy.name}</h1>
-        <img src={toy.imageUrl} alt="toy-image" width="300" height="300" />
+        <img src={toy.image_url} alt="toy-image" width="300" height="300" />
```

Comment:
Think you'll want to keep things consist here, in some places it's `image_url` and others it's `imageUrl`.

There's a few options here:
1) Tweak the API so that it accepts `imageUrl` (camelCase) and any responses it returns `imageUrl` (camelCase).  Should be an easy way to convert from camelCase to snake_case in Ruby/Rails.
2) Tweak the front-end to use `imageUrl` everywhere, except when you're fetching the toys (convert `image_url` to `imageUrl` before you dispatch the FETCH_TOYS action).  Also, make sure anywhere data is being returned from the api you convert `image_url` to `imageUrl`.  A bit of a pain in the butt.
3) Go back to just using `image_url` everywhere.  Even though camelCase is the preferred way in JavaScript, I'd sooner have things consistent then see both `image_url` and `imageUrl` used in different places.

Reply:
Oh okay, I am glad that I can just leave it as `image_url`.  I went ahead and changed toyform back to` image_url` as well. In the future, I will ensure that I start with camelCase.

Reply:
Looks like you chose option #3 (saw a commit in your other PR), so things should go back to being consistent. 

My suggestion here is to watch your knee-jerk reactions, like when I mentioned using camelCase in JS and you went and changed a bunch of places to use imageUrl-- but hadn't quite through through all of the ramifications of doing so.  It's easy when refactoring to break things, the best thing you can do is just put some thought into what you're doing and think about how it might affect other things.

Obviously, having automated testing in place before refactoring is a huge tool, in that it should start failing tests if you change something that causes different results.

Reply:
Oh for sure. I was going to change it throughout the app, but I wanted to get your opinion on how I changed the` image_url` to `imageUrl` and I only wanted to change one file at a time so I started with toyform first. I will make sure to use camelCase in the beginning next time haha!

Reply:
With something like changing `image_url` to `imageUrl`, you can't simply change it in one file at a time because it's used in more than just a single file.  By changing it in a single file, it's either going to not display the appropriate data (looking for `imageUrl` when the field in the state is `image_url`) or not save the appropriate data (sending `imageUrl` to API, when it's expecting `image_url`).  In other words, chances are it will break the app in some way (and in production environments, customers tend not to be very happy when we break an app). :)